tslib.es6.js

```diff
@@ -209,13 +209,20 @@ var __setModuleDefault = Object.create ? (function(o, v) {
     o["default"] = v;
 };
 
-export function __importStar(mod) {
-    if (mod && mod.__esModule) return mod;
-    var result = {};
-    if (mod != null) for (var k in mod) if (k !== "default" && Object.prototype.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
-    __setModuleDefault(result, mod);
-    return result;
-}
+export var __importStar = function (mod) {
+    var cache = typeof WeakMap === "function" ? new WeakMap() : null;
+    __importStar = function (mod) {
+        if (mod === null || (typeof mod !== "object" && typeof mod !== "function")) return { default: mod };

```

Comment:
nit: Parens aren't needed here, since `&&` has higher precedence than `||`.
```suggestion
        if (mod === null || typeof mod !== "object" && typeof mod !== "function") return { default: mod };
```